optimize/upgrade/src/main/java/io/camunda/optimize/upgrade/main/UpgradeProcedure.java

```diff
@@ -9,6 +9,7 @@
 
 import static io.camunda.optimize.upgrade.steps.UpgradeStepType.REINDEX;
```

Comment:
there is a localisation key called `invalidEventProcessState`. Can this be deleted?

Reply:
Got that in the `Fe follow ups` section, didnt want to touch anything before knowing its not referred to from FE anymore